docs/articles/new-expensify/connections/Disconnect-QuickBooks-Online-from-Expensify.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Disconnect QuickBooks Online from Expensify
+description: Remove QuickBooks Online integration with Expensify
+---
+<div id="new-expensify" markdown="1">
+
+{% include selector.html values="desktop, mobile" %}
```

Comment:
@ren-jones i don't think we need platform selector here. Please remove code related to it

Reply:
I've been including the selector on all articles for now. But we will absolutely be deciding soon what we want to do about the platform selectors now that we're getting closer to having Classic and New Expensify docs in order

Reply:
@ren-jones there is info only about `desktop` here. So we can remove this
```suggestion
```